starforge/execution/docker.py

```diff
@@ -0,0 +1,51 @@
+"""
+"""
+from __future__ import absolute_import
+
+from subprocess import check_call
+try:
+    from subprocess import check_output
+except ImportError:
+    from ..util import check_output
```

Comment:
Both squeeze and EL6 use Python 2.6.

Probably not a ton of squeeze boxes out there anymore, but plenty of EL6es.